.babelrc

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
Big whitespace,  maybe bring in line with rest of project (which appears to be four spaces)